packages/frontend/web-ui/src/scss/auth/register/_page.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../base/variables' as var;
+
+.register-form-grid {
+  background-color: var(--cg-text-primary);
+  border-radius: 10px;
+  text-align: center;
+  color: var(--cg-bkg-primary-light);
+  padding: 3rem;
+  margin: 0rem 2rem;
+}
+
+@media (min-width: var.$sm) {
+  .register-form-grid {
+    margin: 0;
+    max-width: 450px;
+  }
+}
+
+.register-title-text-position {
+  padding-bottom: 1.5rem;
+}
+
+.form-register-error {
+  margin-top: 1.5rem;
+  text-align: left;
+}
+
+.form-register-success {
+  margin-top: 1.5rem;
+  text-align: left;
+}
+
+.register-form-grid .form-fieldset-border-color {
```

Comment:
`form-fieldset-border-color`has the same issue here.

Reply:
Done! I've renamed the CSS class name.